lib/template/search-geojson.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+$aFilteredPlaces = array();
+
+foreach($aSearchResults as $iResNum => $aPointDetails)
+{
+
+    $aPlace = array(
+            'place_id'=>$aPointDetails['place_id'],
+            'licence'=>"Data © OpenStreetMap contributors, ODbL 1.0. http://www.openstreetmap.org/copyright",
```

Comment:
I believe that GeoJSON allows to put that on the FeatureCollection? If so, that would be better than repeating it for every feature.